forem/forem
# Update admin welcome checklist

## What type of PR is this? (check all applicable)

## Description
This updates the admin welcome checklist according to specifications laid out in https://github.com/forem/internalCommunity/issues/153

![image](https://user-images.githubusercontent.com/17884966/125998317-c08e661d-eb6e-482a-9552-a0ae4e4c358d.png)

## QA Instructions, Screenshots, Recordings
1. Go to localhost:3000/admin
2. Check that all the links work

### UI accessibility concerns?
Nope, just basic text and links

## Added/updated tests?
- [x] No, and this is why: change in copy

## [Forem core team only] How will this change be communicated?
- [x] communicated to admins through /admin
